mmcv/runner/hooks/logger/tensorboard.py

```diff
@@ -46,9 +46,11 @@ def log(self, runner):
             record = runner.log_buffer.output[var]
             if isinstance(record, str):
                 self.writer.add_text(tag, record, runner.iter)
+            elif isinstance(record, torch.Tensor):  # logs images
+                if len(record.shape) == 4:
```

Comment:
`add_image` accepts the input tensor with 3 dims by default, see [here](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_image).

Reply:
It may also be a numpy array, and a common case is that we visualize the results on an image with numpy.